src/WebServer_DevicesPage.ino

```diff
@@ -395,7 +395,7 @@ void handle_devicess_ShowAllTasksTable(byte page)
     {
       LoadTaskSettings(x);
       struct EventStruct TempEvent;
-      byte spi_gpios[3] = {static_cast<byte>(-1)};
+      uint8_t spi_gpios[3] {0xFF, 0xFF, 0xFF};  // Keep length at least at 3 or the tests below will fail
```

Comment:
You're still comparing it to -1, so why not use int8_t?

Reply:
My eyes, fingers and brain weren't wired correctly, between reading your earlier review and making the correction.